src/app/creative-team/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,82 @@
+<div fxLayout="row" fxLayoutAlign="start start" fxLayoutGap="20px" class="wrapper px-20 pt-30">
+  <mat-card fxFlex="67" class="card--shadow">
+    <mat-card-header class="form">
+      <mat-card fxFlex="100" class="form__header user--color">
+        <p class="bg--color">Edit Profile</p>
+        <p class="text--color">Complete your profile</p>
+      </mat-card>
+    </mat-card-header>
+    <mat-card-content class="form__content mt-60">
+      <div fxLayout="row" fxLayoutAlign="space-evenly center" fxLayoutGap="18px">
+        <mat-form-field fxFlex="45">
+          <mat-label>Company(disabled)</mat-label>
+          <input matInput disabled="all">
+        </mat-form-field>
+        <mat-form-field fxFlex="25">
+          <mat-label>Username</mat-label>
+          <input matInput>
+        </mat-form-field>
+        <mat-form-field fxFlex="30">
+          <mat-label>Email address</mat-label>
+          <input matInput>
+        </mat-form-field>
+      </div>
+      <div fxLayout="row" fxLayoutAlign="space-evenly center" fxLayoutGap="18px">
+        <mat-form-field fxFlex="50">
+          <mat-label>First Name</mat-label>
+          <input matInput>
+        </mat-form-field>
+        <mat-form-field fxFlex="50">
+          <mat-label>Last Name</mat-label>
+          <input matInput>
+        </mat-form-field>
+      </div>
+      <mat-form-field class="form__area">
+        <mat-label>Adress</mat-label>
+        <input matInput>
+      </mat-form-field>
+      <div fxLayout="row" fxLayoutAlign="space-evenly center" fxLayoutGap="18px">
+        <mat-form-field fxFlex="33">
+          <mat-label>City</mat-label>
+          <input matInput>
+        </mat-form-field>
+        <mat-form-field fxFlex="33">
+          <mat-label>Country</mat-label>
+          <input matInput>
+        </mat-form-field>
+        <mat-form-field fxFlex="33">
+          <mat-label>Postal Code</mat-label>
+          <input matInput>
+        </mat-form-field>
+      </div>
+      <p>About Me</p>
+      <mat-form-field class="form__area">
+        <mat-label>Lamborghini Mercy, Your chick she so thirsty,i am in that two seat Lambo.</mat-label>
+        <textarea matInput></textarea>
+      </mat-form-field>
+    </mat-card-content>
+    <mat-card-footer fxLayoutAlign="flex-end center" class="header__button mb-14 mr-8">
+      <button mat-button class="form__button button--design user--color bg--color font-size-12 py-14 px-30 br-4">UPDATE
+        PROFILE
+      </button>
+    </mat-card-footer>
+  </mat-card>
+  <!--  avatar card-->
+  <div fxFlex="33">
+    <mat-card fxLayout="column" fxLayoutAlign="center center" class="profile card--shadow">
+      <mat-card-header class="profile__header">
+        <div mat-card-avatar class="profile__image"></div>
```

Comment:
rather used the image tag and pass the image src here and width and height, when you pass the in the div you are passing as background image not as avatar image